WikiDrips/WikiDocViewController.swift

```diff
@@ -11,8 +11,8 @@ import WebKit
 
 class WikiDocViewController: UIViewController, WKUIDelegate {
 
-    var webView: WKWebView?
-    var searchText: String?
+    private var webView: WKWebView?
+    public var searchText: String?
```

Comment:
Don't need the public keyword here since that's only for visibility outside of the module (which in this case is the app). `public` is generally used only in Frameworks.